lms/static/sass/features/_course-experience.scss

```diff
@@ -1,3 +1,66 @@
+// Course call to action message
+.course-message {
+  .message-author {
+    display: inline-block;
+    width: $baseline*7/2;
+    border-radius: $baseline*7/4;
+    border: 1px solid $lms-border-color;
+
+    @media (max-width: 880px) {
```

Comment:
Nit: I may be misunderstanding this, but can you make the mobile setting be the default, and then have larger sizes allow more to be shown? This is more in line with mobile-first and progressive enhancement.